sencha-workspace/packages/slate-cbl/src/view/teacher/skill/OverrideWindow.js

```diff
@@ -0,0 +1,84 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.teacher.skill.OverrideWindow', {
+    extend: 'Ext.window.Window',
+    xtype: 'slate-cbl-teacher-skill-overridewindow',
+    requires: [
+        'Slate.cbl.view.teacher.skill.OverrideWindowController',
+
+        'Ext.layout.container.Fit',
+        'Ext.form.Panel',
+        'Ext.tab.Panel',
+        'Ext.grid.Panel',
+        'Ext.grid.feature.Grouping',
+        'Ext.grid.column.Action',
+        'Ext.form.field.Text',
+        'Ext.form.field.TextArea',
+        'Ext.form.field.ComboBox',
+        'Ext.form.field.Checkbox',
+        'Ext.data.ChainedStore'
+    ],
+
+    controller: 'slate-cbl-teacher-skill-overridewindow',
+    
+    config: {
+        student: null,
+        newLevel: null
+    },
+
+    title: 'Override',
+    width: 600,
+    constrainHeader: true,
+    layout: 'fit',
+
+    items: {
+        reference: 'form',
+        
+        autoScroll: true,
+        bodyPadding: 16,
+        defaults: {
+            anchor: '75%',
+            allowBlank: false,
+            msgTarget: 'side',
+            selectOnFocus: true,
+            labelAlign: 'right',
+            labelWidth: 150
+        },
+        items: [{
+            reference: 'overrideTitle',
+            name: 'title',
+            
+            xtype: 'component'
+        }, {
```

Comment:
that space between }, { shouldn't be there, and the following lines are indented incorrectly